js/scripts.js

```diff
@@ -0,0 +1,15 @@
+$(function() {
+	console.log('DOM loaded');
+
+	$('span:even').css('color', 'red');
+
+	var paragraphs = $('p');
+	paragraphs.each(function(index, element) {
+    	var button = '<button class="btn" data-tmp="' + index + '">Click here</button>';
+    	$(element).append(button);
```

Comment:
popraw wciecia wewnatrz `each` (najlepiej jak ujednolicisz typ wciec, w sublime jest taka opcja jak `view -> indentation -> convert indentation to ...`